### PR TITLE
Modularize ble

### PR DESCRIPTION
**Pull request by elear**

### CODE REVIEW

File: gateway/database.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 """
 
 Configures Flask with SQLAlchemy for database
-operations and initializes a session.
+perations and initializes a session.
```
Comment (rohitmohan96) [inline gateway/database.py:9]: This doesn't look right.

File: gateway/models.py [hunk 2]
```diff
@@ -1,57 +1,25 @@
-# Copyright (c) 2023, Cisco Systems, Inc. and/or its affiliates.
+# Copyright (c) 2023-2024, Cisco Systems, Inc. and/or its affiliates.
```
Comment (rohitmohan96) [inline gateway/models.py:1]: Do we need to change this in every file? We can probably do it with `reuse`.

Comment (elear) [reply gateway/models.py:1]: Not at this time.

File: gateway/database.py (hunk 1 above)
Comment (elear) [reply gateway/database.py]: Corrected.

**Review by rohitmohan96 — APPROVED** [review]

**Pull request merged**